orderbook/openapi.yml

```diff
@@ -244,8 +244,11 @@ paths:
           description: Token non-existent or not connected to native token
 components:
   schemas:
+    TransactionHash:
+      description: 32 byte digest encoded as a hex with `0x` prefix.
+      example: "0xd51f28edffcaaa76be4a22f6375ad289272c037f3cc072345676e88d92ced8b5"
```

Comment:
PS - this is the transaction hash of our biggest COW yet. We may want to update this to the new contracts at some point.

https://etherscan.io/tx/0xd51f28edffcaaa76be4a22f6375ad289272c037f3cc072345676e88d92ced8b5